addons/mail_group/tests/test_mail_group_message.py

```diff
@@ -122,3 +122,38 @@ def test_email_empty_from(self):
 
         mails = self.env['mail.mail'].search([('subject', '=', 'Test subject')])
         self.assertEqual(len(mails), 0, "Email should not be delivered when no email is specified")
+
+    def test_batch_emails(self):
```

Comment:
Better order tests alphabetically in a given class so that execution order matches test order (easier to debug when having several failing tests ^^ ).